pkg/render/common/elasticsearch/decorator.go

```diff
@@ -139,7 +140,7 @@ func DefaultVolume() corev1.Volume {
 		Name: "elastic-ca-cert-volume",
 		VolumeSource: v1.VolumeSource{
 			Secret: &v1.SecretVolumeSource{
-				SecretName: PublicCertSecret,
+				SecretName: esgateway.EsGatewayElasticPublicCertSecret,
```

Comment:
Using the esgateway package directly here makes me a little wary for two reasons:
1. I don't know that this decorator needs to know about a gateway
2. Importing this means you can't import in the other direction (i.e. import elasticsearch items into the gateway package) because you'd have a cyclic dependency

I'd personally strive for low coupling and use dependency injection to pass the name through, or just redefine the default value here.